### PR TITLE
Sort items for translator

### PR DESCRIPTION
**Pull request by retorquere**
In response to https://forums.zotero.org/discussion/60375?page=1#Item_6

### CODE REVIEW

File: chrome/content/zotero/xpcom/translation/translate_item.js [hunk 1]
```diff
@@ -896,4 +899,4 @@ Zotero.Translate.ItemGetter.prototype = {
 		return obj;
 	}
 }
-Zotero.Translate.ItemGetter.prototype.__defineGetter__("numItemsRemaining", function() { return this._itemsLeft.length });
\ No newline at end of file
+Zotero.Translate.ItemGetter.prototype.__defineGetter__("numItemsRemaining", function() { return this._itemsLeft.length });
```
Comment (dstillman) [inline chrome/content/zotero/xpcom/translation/translate_item.js]: Can you `git checkout -p` this line?

Comment (retorquere) [inline chrome/content/zotero/xpcom/translation/translate_item.js]: I edited the file online on github, but I've done a local checkout and tried both `git checkout -p` and `git checkout -p chrome/content/zotero/xpcom/translation/translate_item.js`, and in both cases it reports back `No changes.`

Comment (dstillman) [inline chrome/content/zotero/xpcom/translation/translate_item.js]: That's for unstaged changes, so you'd have had to do a `git reset HEAD^` first, remove the line, commit again, and force push. Anyway, not a big deal — thanks.

Comment (retorquere) [inline chrome/content/zotero/xpcom/translation/translate_item.js]: My pleasure. Should I submit a similar change for 4.0, or is 4.0 in maintenance mode?

Comment (dstillman) [inline chrome/content/zotero/xpcom/translation/translate_item.js]: It's in maintenance mode, but if there's a simple patch we can take it. (Not as big a deal when we can just skip the changes on merge conflict.)

**Pull request merged**